.github/workflows/release.yml

```diff
@@ -39,27 +39,28 @@ jobs:
     strategy:
       matrix:
         os: ['macos-latest', 'windows-latest', 'ubuntu-18.04']
-        python-version: ['3.5', '3.6', '3.7']
+        py-version: ['3.5', '3.6', '3.7']
+        tf-version: ['2.1.0', '2.2.0rc1']
       fail-fast: false
     steps:
       - if: github.event_name == 'push'
         run: echo "::set-env name=NIGHTLY_FLAG::--nightly"
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: ${{ matrix.py-version }}
       - name: Build wheels
-        if: github.event_name == 'push' || github.event_name == 'release' || matrix.python-version == env.OLDEST_PY_VERSION
+        if: (matrix.os != 'ubuntu-18.04' || matrix.tf-version != '2.2.0rc1') && (github.event_name == 'push' || github.event_name == 'release' || matrix.py-version == env.OLDEST_PY_VERSION)
```

Comment:
Could you explain why we're skipping ubuntu+2.2?
![image](https://user-images.githubusercontent.com/18154355/77930393-d4cbf600-7278-11ea-8108-70daa2c32f31.png)

Reply:
Because we don't have the docker image to build it yet :)

Reply:
Ah gotcha. So the docker container for 2.2.0 won't have any changes other than pre-installed TF. GCC and CUDA versions are the same so removing this should work.

I guess the question is how we want to manage this. Would you prefer we wait until a 2.2.0 docker image is released to build/test so that it is explicit?

Reply:
I prefer to err on the side of caution concerning the CI. Breaking the master branch means nobody can make new pull request and have the CI displaying meaningful results. I know it's very unlikely that we break it by using the 2.1.0 image. But there is very little benefit with working on this before the official image is released. I'd prefer to merge this and when the official image is released, we remove the exception in the yaml and fix potential incompatibilities. This new image will have python 3.8, will be possibly bigger and burst our disk size limit in github actions, I'd prefer we do it one step at a time.

Reply:
Agree